sumatra/recordstore/django_store/models.py

```diff
@@ -80,6 +80,9 @@ def __unicode__(self):
     def last_updated(self):
         return self.record_set.all().aggregate(models.Max('timestamp'))["timestamp__max"] or datetime(1970, 1, 1, 0, 0, 0)
 
+    def get_mainfiles(self):
```

Comment:
minor point of style, but this should be `get_main_files` to match the attribute name `main_file`